File: Week 6/mario.py
def main():
    height = 0
    while height < 1 or height > 8:
        height = int(input("Height: "))
    pyramid(height, 0)
def pyramid(height, space):
    if height == 0:
        return
    pyramid(height - 1, space + 1)
    print(" " * space, end="")
    print("#" * height, end="")
    print("  ", end="")
    print("#" * height)
if __name__ == "__main__":
    main()

File: Week 1/mario.c
#include <stdio.h>
#include <cs50.c>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int times = 1; times <= height; times++)
    {
        int space = height - times;
        for (int space_print = 0; space_print < space; space_print++)
        {
            printf(" ");
        }
        for (int hash = 0; hash < times; hash++)
        {
            printf("#");
        }    
        printf("  ");
        for (int hash = 0; hash < times; hash++)
        {
            printf("#");
        }   
        if (times != height)
        {   
            printf("\n");
        }

    }

}